app/src/editor/dashboard/index.jsx

```diff
@@ -197,7 +198,13 @@ const DashboardLoader = withRouter(withErrorBoundary(ErrorComponentView)(class D
 
     load = async (dashboardId) => {
         this.setState({ isLoading: dashboardId })
-        const newDashboard = await services.loadDashboard({ id: dashboardId })
+        let newDashboard
+        try {
+            newDashboard = await services.loadDashboard({ id: dashboardId })
+        } catch (error) {
+            if (this.unmounted || this.state.isLoading !== dashboardId) { return }
```

Comment:
`isLoading` is not a good name for an id, haha. ;)

Reply:
it "is loading this id" 🤷‍♀

Reply:
This section needs gutting and given the same treatment as canvas loading, the code will go away eventually.